src/main/java/studio/magemonkey/fusion/cfg/Cfg.java

```diff
@@ -1,182 +1,125 @@
 
         return cfg;
     }
 
     private static void addDefs(FileConfiguration cfg) {
-        cfg.addDefault("recursive_level_formula", recursive);
-        cfg.addDefault("final_level_mod", finalMod);
-        cfg.addDefault("data_save_interval", dataSaveInterval); //Auto save every 10 minutes
-        cfg.addDefault("forget.penalty", forgetPenalty);
-
-        HashMap<Character, ItemStack> items = new HashMap<>();
-        items.put('0', ItemBuilder.newItem(Material.STONE).durability(ItemColors.BLACK).build());
-        items.put('>', ItemBuilder.newItem(Material.BOOK).name("Next page").build());
-        items.put('<', ItemBuilder.newItem(Material.BOOK).name("Prev page").build());
-        InventoryPattern ip =
-                new InventoryPattern(new String[]{"=========", "=========", "=========", "=========", "=========", "<0000000>"},
-                        items);
-        ItemStack item = new ItemStack(Material.BLACK_STAINED_GLASS_PANE/*, 1, (short) 15*/);
-//        CraftingTable a = new CraftingTable("forge", "Forge inventory name", DarkRiseEconomy.getItemsRegistry().getItems().iterator().next(), ip, item/*new ItemStack(Material.BLACK_STAINED_GLASS_PANE)*/, 0, 0);
-//
-//        a.addRecipe(new Recipe("test",
-//                Arrays.asList(new RecipeEconomyItem("testItem", 5),
-//                        new RecipeCustomItem(new ItemStack(Material.COOKIE), 2, true)),
-//                new RecipeEconomyItem("resultItem", 4), 0, 0, 0));
-        CraftingTable b = new CraftingTable("craft",
-                "Craft inventory name",
-                new VanillaProvider.VanillaItemType(Material.PAPER),
-                ip,
-                item/*new ItemStack(Material.BLACK_STAINED_GLASS_PANE)*/,
-                0,
-                0);
-        List<Map<String, Object>> list = new ArrayList<>(3);
-//        list.add(a.serialize());
-        list.add(b.serialize());
-        cfg.addDefault("types", list);
+        // addDefault did not add it to config, so here is a work around
+        if(!cfg.isSet("recursive_level_formula")) cfg.set("recursive_level_formula", recursive);
+        if(!cfg.isSet("final_level_mod")) cfg.set("final_level_mod", dataSaveInterval);
+        if(!cfg.isSet("data_save_interval")) cfg.set("data_save_interval", dataSaveInterval);
+        if(!cfg.isSet("forget.penalty")) cfg.set("forget.penalty", forgetPenalty);
+        if(!cfg.isSet("crafting_queue")) cfg.set("crafting_queue", craftingQueue);
+        if(!cfg.isSet("finished_message_interval")) cfg.set("finished_message_interval", 300);
     }
 
     public static void init() {
-        map.clear();
-        guiMap.clear();
-        File              file = new File(Fusion.getInstance().getDataFolder(), "config.yml");
-        FileConfiguration cfg  = new YamlConfiguration();
+        File file = new File(Fusion.getInstance().getDataFolder(), "config.yml");
+        FileConfiguration cfg = new YamlConfiguration();
 
         if (!file.exists()) {
```

Comment:
Should we replace this block with a `getConfig` call? It looks like this code I the exact same

Reply:
You are right, I have updated this.